### ISSUE TITLE
Allow the user to choose rather than typing in yes/no for git installation

### ISSUE THREAD

**Issue by jamesgeorge007** [madlabsinc/mevn-cli#91]
Currently [gitRepo.js](https://github.com/madlabsinc/mevn-cli/blob/master/lib/commands/deploy/gitRepo.js) validates whether `git` is installed within the user's machine and prompts to type in (Y/N) such that if it should continue with the installation or not.

> It should be replaced with a confirm dialog as supported by `inquirer`

**Comment by amitsin6h** [comment]
I would like to work on this feature. Please assign me

**Comment by anto-christo** [comment]
Assigned!! All the best :+1:

**Comment by ajomadlabs** [comment]
@amitsin6h How's the progress

**Comment by amitsin6h** [comment]
@ajomadlabs I working today itself. While working I found something strange.
When I did mevn init <appname>
after that I got the below list of commands
but what I found is mevn-cli does not works only mevn works.
I just want to know is it changed from mevn-cli to mevn or something is wrong with my setup

**Comment by amitsin6h** [comment]
@anto-christo @ajomadlabs  to test the git installation I removed the git from my system. After that I run `mevn init <appname>`
below are the results 
```

➜  mevn init testapp



  __  __                             _ _ 
 |  \/  | _____   ___ __         ___| (_)
 | |\/| |/ _ \ \ / / '_ \ _____ / __| | |
 | |  | |  __/\ V /| | | |_____| (__| | |
 |_|  |_|\___| \_/ |_| |_|      \___|_|_|
                                         
Tool for mevn stack.
? Please select one basic
exec: /bin/sh: 1: git: not found

mv: cannot stat 'mevn-boilerplate': No such file or directory
exec: mv: cannot stat 'mevn-boilerplate': No such file or directory

Fetching the boilerplate ⠦

```

Do I have to check if the user does not have git installed in his system then ask user to install git with yes /  no option ? 
Please let me know .

**Comment by amitsin6h** [comment]
@jamesgeorge007 @ajomadlabs  sorry for the trouble. I solved it after latest code merger from upstream branch.

**Comment by amitsin6h** [comment]
@anto-christo @ajomadlabs I am working the latest code now.

I removed the git from my system using 'sudo apt-get remove git'
Then I tried to create the project using 'mevn init <appname>'
But below is the error that came
![Screenshot from 2019-03-14 13-15-35](https://user-images.githubusercontent.com/12906090/54339827-f18c9000-465b-11e9-8db7-0627fe0bbd17.png)

Second time I installed the git in my system using 'sudo apt-get install git'
Then I tried to create the project using 'mevn init <appname>'
It created the project

I guess now I have to handle the exception and then ask the user for the installation of git. So that it does not create any problem while creating the project.

My OS details
'Linux juno 4.15.0-39-generic #42-Ubuntu SMP Tue Oct 23 15:48:01 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux'
